src/import-data/import-utils.ts

```diff
@@ -14,18 +33,24 @@ export type ImportOptions = {
  * Can the user can import data into a workspace?
  *
  * @param importOptions
+ * @param importOptions.cloudPlatform - Cloud platform required for the import.
  * @param importOptions.isProtectedData - Is the source data protected.
  * @param importOptions.requiredAuthorizationDomain - Authorization domain required for the source data.
  * @param workspace - Candidate workspace.
  */
 export const canImportIntoWorkspace = (importOptions: ImportOptions, workspace: WorkspaceWrapper): boolean => {
-  const { isProtectedData, requiredAuthorizationDomain } = importOptions;
+  const { cloudPlatform, isProtectedData, requiredAuthorizationDomain } = importOptions;
 
   // The user must be able to write to the workspace to import data.
   if (!canWrite(workspace.accessLevel)) {
     return false;
   }
 
+  // If a cloud platform is required, the destination workspace must be on that cloud platform.
+  if (cloudPlatform && getCloudProviderFromWorkspace(workspace) !== cloudPlatform) {
```

Comment:
When I was working in Rawls, I was dealing with two different sets of enums for snapshot vs. workspace cloudPlatform -- just want to confirm that the types here in the UI code are somehow converged to the same enum/type?

Reply:
Yes, they're both converted the `CloudProvider` type (`AZURE` and `GCP`).